docusaurus/docs/dev-docs/database-migrations.md

```diff
@@ -19,6 +19,8 @@ Strapi automatically detects migration files and run them once at the next start
 
 :::warning
 Currently Strapi does not support down migrations. This means that if you need to revert a migration, you will have to do it manually. It is planned to implement down migrations in the future but no timeline is currently available.
+
+Strapi will delete any unknown tables without warning. This means that database migrations can only be used to keep data when changing the Strapi schema.
```

Comment:
As this is unrelated to the previous sentence, we should add bullet points to lines 21 and 23.